kindle.py

```diff
@@ -14,6 +14,7 @@
 import time
 import urllib
 from http.cookies import SimpleCookie
+from faker import Faker
```

Comment:
I think we'd better do not use faker.

It is quite a big package. just one random list may be enough.
what do you think @frostming